build.gradle

```diff
@@ -58,11 +58,13 @@ ext {
             annotation: 'androidx.annotation:annotation:1.1.0',
             appcompat: 'androidx.appcompat:appcompat:1.1.0',
             core: 'androidx.core:core:1.3.0',
-            navfragment: "androidx.navigation:navigation-fragment:$nav_version",
-            navfragmentktx: "androidx.navigation:navigation-fragment-ktx:$nav_version",
-            navui: "androidx.navigation:navigation-ui:$nav_version",
-            navuiktx: "androidx.navigation:navigation-fragment-ktx:$nav_version",
-            navfeatfragment: "androidx.navigation:navigation-dynamic-features-fragment:$nav_version",
+            nav: [
+                    fragment: "androidx.navigation:navigation-fragment:$navigationversion",
```

Comment:
Single quotes for all of these as well, not double quotes.

Reply:
I need to use double quotes in order to reference the $navigationversion variable (with single quotes, it sees $navigationversion as part of the string rather than a variable).

Reply:
Oh you're right, got it!